dotcom-rendering/src/model/enhanceCards.ts

```diff
@@ -140,10 +140,17 @@ export const enhanceCards = (
 			? enhanceTags(faciaCard.properties.maybeContent.tags.tags)
 			: [];
 
+		// The URL parameter on a Snap header will be a link to the Snap itself, there is a second href
+		// property which contains what the snap is actually linking to.
+		const url =
```

Comment:
NavList tends to use a lot of Snaps which is how I noticed this.

Reply:
Could we use JSDoc here?